docs/sources/setup-grafana/installation/debian/index.md

```diff
@@ -38,112 +28,136 @@ If you install from the APT repository, then Grafana is automatically updated ev
 
 > **Note:** Grafana Enterprise is the recommended and default edition. It is available for free and includes all the features of the OSS edition. You can also upgrade to the [full Enterprise feature set](https://grafana.com/products/enterprise/?utm_source=grafana-install-page), which has support for [Enterprise plugins](https://grafana.com/grafana/plugins/?enterprise=1&utcm_source=grafana-install-page).
 
-#### To install the latest release:
+### Before you begin
 
-```bash
-sudo apt-get install -y apt-transport-https
-sudo apt-get install -y software-properties-common wget
-sudo wget -q -O /usr/share/keyrings/grafana.key https://apt.grafana.com/gpg.key
-```
+To follow these instructions, you need:
 
-Add this repository for stable releases:
+- xxx
+- xxx
 
-```bash
-echo "deb [signed-by=/usr/share/keyrings/grafana.key] https://apt.grafana.com stable main" | sudo tee -a /etc/apt/sources.list.d/grafana.list
-```
+### Steps
 
-Add this repository if you want beta releases:
+Complete the following steps to install Grafana from the APT repository:
 
-```bash
-echo "deb [signed-by=/usr/share/keyrings/grafana.key] https://apt.grafana.com beta main" | sudo tee -a /etc/apt/sources.list.d/grafana.list
-```
+1. To install the latest release, run the following commands:
```

Comment:
@DanCech , each of the other steps in this section begin with "To add a repository...". Is it the same case here? Should this step be "To add a latest release repository, run..."?

Reply:
```suggestion
1. To install required packages and download the Grafana repository signing key, run the following commands:
```